tools/kete.py

```diff
@@ -543,13 +543,19 @@ def _log_drawings_available(self, device):
 
 
 class Fetcher(Worker):
-    def __init__(self, manager, args):
+    def __init__(self, manager, args, config):
         super(Fetcher, self).__init__(manager)
         self.device = None
         self.timestamps = None
         address = args.address
         index = args.index
 
+        if not address in config:
+            config[address] = {}
+
+        self.orientation = int(config[address].get('Orientation', 0))
```

Comment:
Can we make this a string instead?
Like `Portrait`, `Landscape`, `Upside-Down-Portrait`, `Upside-Down-Landscape` (not usre about how to properly name the last 2).